lib/galaxy/config/schemas/config_schema.yml

```
@@ -3764,14 +3764,15 @@ mapping:
           Activate this only if you have setup a Celery worker for Galaxy and you have
           configured the `celery_conf` option below. Specifically, you need to set the
           `result_backend` option in the `celery_conf` option to a valid Celery result
-          backend URL.
+          backend URL. By default, Galaxy uses an SQLite database at '<data_dir>/results.sqlite' for storing task results.
+          Please use a more robust backend for production setups like Redis.
```

```suggestion
          Please use a more robust backend for production setups (e.g. Redis).
```
Otherwise,  think it reads "production setups like Redis"

---

Or even clearer:

```suggestion
          Please use a more robust backend (e.g. Redis) for production setups.
```

---

Out of interest: what does "robust" mean here? Why would sqlite not fit this term?

---

SQLite doesn`t scale very well for big instances because of concurrency access, maybe for small ones, it is completely fine.